src/app/login/logInForm.tsx

```diff
@@ -0,0 +1,58 @@
+"use client";
+
+import React, { useState } from "react";
+import { useForm } from "react-hook-form";
+import { loginUser } from "../../api/loginApi";
+import { useRouter } from "next/navigation";
+
+import Input from "@/components/inputs/Input";
+
+type LoginFormInputs = {
+  email: string;
+  password: string;
+};
+
+export function LogInForm() {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors, isSubmitting },
+  } = useForm<LoginFormInputs>();
+  const [formError, setFormError] = useState("");
+  const router = useRouter();
+
+  const onSubmit = async (data: LoginFormInputs) => {
+    setFormError("");
+    try {
+      await loginUser(data);
+      router.push("/");
+    } catch (err: any) {
+      setFormError(err.message);
+    }
+  };
+
+  return (
+    <div>
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <Input
+          id="email"
+          label="Email"
+          type="email"
+          {...register("email", { required: "Email is required" })}
```

Comment:
WDYT about adding some simple validation for email here? It's a login form, so we can assume that the user already has a valid account when trying to log-in, but it'd possibly prevent some simple mistakes.